code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -152,6 +152,13 @@
 	desc = "A syndicate belt designed to be used by boarding parties.  Its style is modeled after the hardsuits they wear."
 	icon_state = "militarybelt"
 	item_state = "military"
+	max_w_class = 4
+
+/obj/item/weapon/storage/belt/military/army

```

Comment:
Army tends to mean land army, rather than military as a whole. So Bawhoppen's probably setting us up for military/naval belts.